src/tracker/tracks_manager.py

```diff
@@ -0,0 +1,98 @@
+from src.tracker.track import Track
+from typing import Dict
+import sqlite3
+import numpy as np
+from src.config.config import TracksManagerConfig
+
+# TODO: make it more maintanable and have map[name_of_table, list[columns]]
+tables: Dict[str, str] = {
+    "tracks": """
+        CREATE TABLE IF NOT EXISTS tracks(
+            id CHAR PRIMARY KEY,
```

Comment:
Nice, I also agree that using the original person_timestamp string is a good key to use across tables